pkgs/games/re3/default.nix

```diff
@@ -0,0 +1,42 @@
+{ branch ? "re3"
+, callPackage
+, fetchFromGitHub
+}:
+
+{
+  re3 = callPackage ./base.nix rec {
+    pname = "re3";
+    version = "12-07-2021";
+
+    src = fetchFromGitHub {
+      repo = "re3";
+      owner = "GTAmodding";
+      rev = "a4b92fe9bec17ad00dc436574f57844c4b8d49f5";
+      sha256 = "5CPgDhUgkB0B5EApwkdhh7qqEbtCE3Fee1CKu3BTGz4=";
+    };
+  };
+
+  miami = callPackage ./base.nix rec {
+    pname = "reVC";
+    version = "12-07-2021";
+
+    src = fetchFromGitHub {
+      repo = "re3";
+      owner = "GTAmodding";
+      rev = "523b23339cd761d934501dcbcc059d131b5a28c4";
+      sha256 = "Cb9C9Gh1VGPPwF2uO82wOdxD2rfaixdkBsClx3OFNvU=";
+    };
+  };
+
+  lcs = callPackage ./base.nix rec {
+    pname = "reLCS";
+    version = "12-07-2021";
+
+    src = fetchFromGitHub {
+      repo = "re3";
+      owner = "GTAmodding";
```

Comment:
```suggestion
      owner = "GTAmodding";
      repo = "re3";
      rev = "a4b92fe9bec17ad00dc436574f57844c4b8d49f5";
      sha256 = "5CPgDhUgkB0B5EApwkdhh7qqEbtCE3Fee1CKu3BTGz4=";
    };
  };

  miami = callPackage ./base.nix rec {
    pname = "reVC";
    version = "12-07-2021";

    src = fetchFromGitHub {
      owner = "GTAmodding";
      repo = "re3";
      rev = "523b23339cd761d934501dcbcc059d131b5a28c4";
      sha256 = "Cb9C9Gh1VGPPwF2uO82wOdxD2rfaixdkBsClx3OFNvU=";
    };
  };

  lcs = callPackage ./base.nix rec {
    pname = "reLCS";
    version = "12-07-2021";

    src = fetchFromGitHub {
      owner = "GTAmodding";
      repo = "re3";
```

Reply:
Is that serious? Just exchanging owner and repo lines?

Reply:
Yes, it should have this order to make it easier to find the repo.

Reply:
Ok, fine